index.html

```diff
@@ -39,11 +39,11 @@ <h2 class="content-heading">・About me</h2>
                 <h2 class="content-heading">・Software and Skill</h2>
                 <div class="content-text">
                     <ul>
-                        <li>PLC and SCADA: TIA Portal V13</li>
-                        <li>PCB Design: Altium, Orcad</li>
-                        <li>Hydraulic and Pneumatic Simulation: Festo FluidSIM</li>
-                        <li>Microcontroller Programming: Keil C</li>
-                        <li>Mechanical Desgin: SolidWorks</li>
+                        <li><strong>Plc and SCADA  :</strong>  TIA Portal V13</li>

```

Comment:
PLC stands for Programmable Logic Controller so it should be all letters uppercased (PLC)